launch/LMS1xx.launch

```diff
@@ -1,6 +1,13 @@
 <launch>
   <arg name="host" default="192.168.1.14" />
+  
+  <!-- start and end units are in degrees to keep with lms internal structure. They will be converted to rad in the node. -->
+  <arg name="start" default="-999.9" />
+  <arg name="end" default="999.9" />
```

Comment:
Hmm, that seems confusing. I'd prefer to use the actual range of the sensor as the default, and it really should be rads and `min_angle`/`max_angle`, like how libgazebo_ros_laser is. I can make these changes and merge it in, though.

One other thing— it looks like you're just culling the data after it comes out of the sensor, rather than actually programming the start/stop into the device itself. Am I mistaken about that?